docs/content/preview/develop/change-data-capture/using-logical-replication/get-started.md

```diff
@@ -300,21 +275,21 @@ Kafka Connect reports some "errors". However, you can safely ignore these warnin
 
 ### View change events
 
-After deploying the YugabyteDB connector, it starts monitoring the `yugabyte` database for data change events.
+After it is deployed, the YugabyteDB connector starts monitoring the `yugabyte` database for data change events.
 
 For this tutorial, you will explore the `dbserver1.public.products` topic.
 
 #### View a change event
 
 Open a new terminal, and use it to start the watch-topic utility to watch the `dbserver1.public.products` topic from the beginning of the topic.
 
-The following command runs the `watch-topic` utility in a new container using the `2.5.2.Final` version of the `debezium/kafka` image:
+The following command runs the watch-topic utility in a new container using the 2.5.2.Final version of the `debezium/kafka` image:
 
 ```sh
 docker run -it --rm --name consumer --link zookeeper:zookeeper --link kafka:kafka debezium/kafka:2.5.2.Final watch-topic -a dbserver1.public.products
 ```
 
-The `watch-topic` utility returns the event records from the `products` table. There will be 200 events, one for each row in the table which was snapshotted. Each event is formatted in JSON, because that is how you configured the Kafka Connect service. There are two JSON documents for each event: one for the key, and one for the value.
+The watch-topic utility returns the event records from the `products` table. There will be 200 events, one for each row in the table which was snapshotted. Each event is formatted in JSON, because that is how you configured the Kafka Connect service. There are two JSON documents for each event: one for the key, and one for the value.
 
 You should see output similar to the following:
 
```

Comment:
output is all on one line, which doesn't look great

Reply:
Removing the file from this PR.